src/azul/terraform.py

```diff
@@ -793,6 +793,18 @@ def tf_config(self, app_name):
         del deployment['lifecycle']
         deployment['triggers'] = {'redeployment': deployment.pop('stage_description')}
 
+        # Instead of specifying this property in the related Terraform
+        # resource, using an AWS API Gateway extension to the OpenAPI
+        # specification lets us avoid maintaining a complicated trigger
+        # dependency between Terraform resources.
+        #
+        rest_api = resources['aws_api_gateway_rest_api'][app_name]
+        assert 'minimum_compression_size' not in rest_api, rest_api.keys()
```

Comment:
```suggestion
        assert 'minimum_compression_size' not in rest_api, rest_api
```

When it's unexpectedly set, you'd want to see what it's set to.